CHANGELOG.md

```diff
@@ -403,6 +406,16 @@ Deprecated features
   ≅-heterogeneous-irrelevanceʳ ↦ ≅-heterogeneous-irrelevantʳ
   ```
 
+* From `Relation.Binary.PropositionalEquality` to `Relation.Binary.PropositionalEquality.Core`:
+  ```agda
+  cong
+  module ≡-Reasoning
+  trans-reflʳ
+  trans-assoc
+  trans-symˡ
+  trans-symʳ
+  ```
```

Comment:
Do we need to mention this in the CHANGELOG? Under the contract that the `Core` modules should never be imported then these changes shouldn't matter?

Reply:
Fair enough.